### PR TITLE
Fix typo and mention taking the odd extension for b_n

### PR DESCRIPTION
**Pull request by JaKXz**
Since a_n usually goes to 0 with the given ICs

### CODE REVIEW

File: Differential Equations.tex [hunk 1]
```diff
@@ -284,10 +284,13 @@ \subsubsection*{Solution}
 \item Plug in the ICs to get $a_n, b_n$.
 
 \begin{gather*}
-a_n = \frac{1}{l} \int_{-l}^{l} f(x) \sin(\frac{n\pi}{l}x) dx \\
+a_n = \frac{1}{l} \int_{-l}^{l} f(x) \cos(\frac{n\pi}{l}x) dx \\
```
Comment (krishnr) [inline Differential Equations.tex:287]: @JaKXz pretty sure this should be sine

File: Differential Equations.tex [hunk 2]
```diff
@@ -284,10 +284,13 @@ \subsubsection*{Solution}
...
 b_n = \frac{1}{n\pi c} \int_{-l}^{l} g(x) \sin(\frac{n\pi}{l}x) dx
 \end{gather*}
-
+Take the odd extension of $g(x)$ and integrate:
```
Comment (krishnr) [inline Differential Equations.tex:290]: @JaKXz also you don't always have to take the odd extension. See A4 - 9.6 - #1

Comment (JaKXz) [inline Differential Equations.tex:290]: that's true. I should've mentioned if `a_n` goes to 0.

File: Differential Equations.tex (hunk 1 above)
Comment (JaKXz) [inline Differential Equations.tex:287]: Interesting... I was confused by this vs. the definition of `T(t)`. Why is it `sine`?

Comment (krishnr) [inline Differential Equations.tex:287]: @JaKXz Plugging in `u(x,0)=f(x)`, you get `T(0) = a_n*1 + b_n*0` and `X(x)=sin(n*pi/x)`. So `f(x) = sum[a_n sin(n*pi/x)]` and so the a_n integral is a `sine` one.

**Pull request closed without merging**